app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="form">
+  <%= flash[:alert] %>
+
+<%= form_tag("/shelters/#{@shelter.id}/#{@review.id}", method: :patch) do %>
+  <label>Title:</label><br>
+  <%= text_field_tag :title, @review.title %><br>
+
+  <label>Rating (out of 5): </label><br>
```

Comment:
Thanks for adding the rating scale! Looks good!

Reply:
Thanks!